Dockerfiles/Dockerfile_Ubuntu_MPI

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:23.10 
+
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y netcdf-bin libnetcdf-c++4-dev libboost-all-dev libeigen3-dev cmake
+RUN apt-get install -y python3-netcdf4
+RUN apt-get install -y libnetcdf-mpi-dev 
+RUN apt-get install -y openmpi-bin libopenmpi-dev 
+
+ENV CPLUS_INCLUDE_PATH /usr/lib/x86_64-linux-gnu/netcdf/mpi/include
```

Comment:
does this still work? I know we took it out of your `.bashrc`. Does it cause any issues in the docker instance?

Reply:
I just tested it. If I remove the line I am back to `nextsim/core/src/ParallelNetcdfFile.cpp:3:10: fatal error: netcdf_par.h: No such file or directory`, with that line it works.